src/lib.rs

```diff
@@ -245,35 +241,35 @@ impl Firebase {
     /// });
     pub fn get_async<F>(&self, callback: F) -> JoinHandle<()>
     where F: Fn(Result<Response, ReqErr>) + Send + 'static {
-        Firebase::request_url_async(&self.url, Method::GET, None, callback)
+        Firebase::request_url_async(self.url.clone(), Method::GET, None, callback)
```

Comment:
If hyper makes us do all these copies, I'd rather not use it.
Even though its probably a negligible performance loss, it feels dirty.
I would be really happy to include hyper as a feature of the crate, so windows people might use this crate with the `hyper` feature (although this is a different feature, so it would be a separate PR).